js/src/components/google-ads-account-card/authorize-ads.js

```diff
@@ -10,18 +10,18 @@ import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
 import useGoogleAuthorization from '.~/hooks/useGoogleAuthorization';
 import AccountCard, { APPEARANCE } from '.~/components/account-card';
 import AppButton from '.~/components/app-button';
-
-const pageName = 'setup-ads';
+import { glaData } from '.~/constants';
 
 /**
  * @param {Object} props React props
  * @param {string} props.additionalScopeEmail
  * @fires gla_google_account_connect_button_click with `{ action: 'scope', context: 'setup-ads' }`
  */
 const AuthorizeAds = ( { additionalScopeEmail } ) => {
+	const nextPageName = glaData.mcSetupComplete ? 'setup-ads' : 'setup-mc';
```

Comment:
💅 Maybe we can replace this string with constants like SETUP_ADS_PAGE and SETUP_MC_PAGE

Reply:
In this case, I think it's no need to declare additional constants. They are not duplicated in the same scope of uses. The `'setup-ads'` below this file belong to another semantic and use of context, except that they happen to be the same value.

In terms of the whole codebase, whether to create shared constants for event tracking properties such as the `context` will be another 📜 topic.